### PR TITLE
handle exceptions of extensions

### PR DESCRIPTION
**Pull request by lanwen**
[Review on Reviewable](https://reviewable.io/reviews/jenkinsci/github-plugin/64)

### CODE REVIEW

File: src/main/java/org/jenkinsci/plugins/github/extension/GHEventsSubscriber.java [hunk 1]
```diff
@@ -122,7 +125,12 @@ public boolean apply(GHEventsSubscriber subscriber) {
         return new Function<GHEventsSubscriber, Void>() {
             @Override
             public Void apply(GHEventsSubscriber subscriber) {
-                subscriber.onEvent(event, payload);
+                try {
+                    subscriber.onEvent(event, payload);
+                } catch (Throwable t) {
+                    LOGGER.error("Subscriber {} failed to process {} hook, skipping...", 
+                            subscriber.getClass().getName(), event, t);
```
Comment (KostyaSha) [inline src/main/java/org/jenkinsci/plugins/github/extension/GHEventsSubscriber.java]: Does subscriber has descriptor and it's name?

Comment (lanwen) [inline src/main/java/org/jenkinsci/plugins/github/extension/GHEventsSubscriber.java]: No, only subscribable has descriptor.

**Comment by KostyaSha** [comment]
:+1:

**Comment by oleg-nenashev** [comment]
:+1:

**Pull request merged**